src/proto/Solana.proto

```diff
@@ -0,0 +1,45 @@
+syntax = "proto3";
+
+package TW.Solana.Proto;
+option java_package = "wallet.core.jni.proto";
+
+message Transfer {
+    bytes private_key = 1;
+    string recipient = 2;
+    uint64 amount = 3;
+}
+
+message Stake {
+    bytes from_private_key = 1;
+    bytes stake_private_key = 2;
+    string vote_pubkey = 3;
+    uint64 amount = 4;
+}
+
+message DeactivateStake {
+    bytes stake_private_key = 1;
+    string vote_pubkey = 2;
+}
+
+message WithdrawStake {
+    bytes stake_private_key = 1;
```

Comment:
can we use `private_key` name instead? but then add comment as staking private key?